src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -26,4 +26,16 @@ public Movie get(Long id) {
     public List<Movie> getAll() {
         return movieDao.getAll();
     }
+
+    @Override
+    public Boolean update(Movie movie) {
+        Movie currentMovie = get(movie.getId());
+        movieDao.update(movie);
+        return currentMovie.equals(get(movie.getId()));
```

Comment:
The `update` method returns a boolean by comparing the updated entity with the one retrieved from the database. This approach might not be reliable if the entity has fields that are not persisted or if the database automatically modifies some fields (e.g., timestamps). Consider returning a boolean based on the success of the update operation itself, rather than comparing objects.